pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -16,14 +16,15 @@ import (
 	"github.com/grafana/grafana/pkg/api/datasource"
 	"github.com/grafana/grafana/pkg/infra/httpclient"
 	glog "github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/infra/tracing"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/plugins"
 	"github.com/grafana/grafana/pkg/services/datasources"
 	"github.com/grafana/grafana/pkg/services/oauthtoken"
 	"github.com/grafana/grafana/pkg/setting"
 	"github.com/grafana/grafana/pkg/util"
 	"github.com/grafana/grafana/pkg/util/proxyutil"
-	"github.com/opentracing/opentracing-go"
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
I see this is imported for the `attribute.Key()` function, but perhaps we could have an alias in the `tracing` package to simplify the dependency graph? I.e. in the tracing package add a `func Key(k string) attribute.Key { ... }` to avoid unnecessary imports.

Reply:
Actually, nevermind, I see this goes well beyond attribute.Key, and feels identical to using opentelemetry directly, so let's keep it as it is now.